android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeFragment.kt

```diff
@@ -71,7 +91,10 @@ class PracticeFragment : Fragment() {
                 return@Observer
             }
 
-            Toast.makeText(context, "Correct!", Toast.LENGTH_SHORT).show()
+            var toastMessage = "Правильно!"
+            if (viewModel.ifHintUsed())
+                toastMessage += "\n Но с подсказкой балл не даётся"

```

Comment:
We should keep Toast messages short, this notion is not necessary.

Reply:
I pondered it and, ok, finally decided to remove this `if`